Source/items.h

```diff
@@ -367,6 +367,11 @@ struct Item {
 		return IsScroll() && _iSpell == spellId;
 	}
 
+	[[nodiscard]] bool KeyAttributesMatch(int32_t seed, _item_indexes itemIndex, uint16_t createInfo)

```

Comment:
Could we also use this in FindGetItem?

Reply:
Yes, also in SyncGetItem and DeltaAddItem. I'd been avoiding updating those call sites because only DeltaAddItem has a "valid" reason to use it and it's to do an equality check, which should use a method wrapping this check and other conditions.